static/admin/index.html

```diff
@@ -1,6 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- Google Tag Manager -->
```

Comment:
This page is only for the CMS, we dont need google analytics here, it wont track anything on the main wiki site.

We should be using the official GA plugin for Docusaurus instead: https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-google-analytics

Reply:
Can that plugin be used to add google tag manager ID instead of analytics ID?

Reply:
Yes there's one for tags as well: https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-google-gtag